### PR TITLE
Oukitel WP8 Pro

### PR DESCRIPTION
**Pull request by EdGeraghty**
Add Oukitel WP8 Pro overlay

Signed-off-by: Ed Geraghty <ed@geraghty.london>

### CODE REVIEW

File: Oukitel/WP8PROEEA/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.oukitel.wp8proeea"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+		android:requiredSystemPropertyValue="+OUKITEL/WP8_Pro_EEA/WP8_Pro*"
```
Comment (phhusson) [inline Oukitel/WP8PROEEA/AndroidManifest.xml:7]: Match all of them, EEA variant should have the same hardware as non-EEA

File: Oukitel/WP8PROEEA/res/values/strings.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="config_defaultListenerAccessPackages">com.ssui.launcher3:com.android.calendar:com.ssui.push:com.ssui.gmodel:com.android.launcher3:com.google.android.setupwizard:com.google.android.apps.restore:com.google.android.projection.gearhead</string>
```
Comment (phhusson) [inline Oukitel/WP8PROEEA/res/values/strings.xml:3]: This MUST be left to the ROM to decide, it's not related to hardware at all.

File: Oukitel/WP8PROEEA/res/values/integers.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="config_doubleTapOnHomeBehavior">2</integer>
```
Comment (phhusson) [inline Oukitel/WP8PROEEA/res/values/integers.xml:3]: Why? It's better left to the ROM.

**Review by phhusson — COMMENTED** [review]
You also need to add the overlay to overlay.mk

File: Oukitel/WP8PROEEA/res/values/strings.xml (hunk 2 above)
Comment (EdGeraghty) [reply Oukitel/WP8PROEEA/res/values/strings.xml]: Done.

**Comment by EdGeraghty** [comment]
@phhusson I have addressed your comments

**Comment by phhusson** [comment]
You also need to remove config_enableAutoPowerModes (see failed test), and you'll be good to go :-)

**Comment by EdGeraghty** [comment]
Merci :)

**Comment by EdGeraghty** [comment]
@phhusson any chance of getting this PR merged?

**Pull request merged**